rebasehelper/tests/test_specfile.py

```diff
@@ -30,18 +30,14 @@
 from typing import List
 
 from rebasehelper.specfile import SpecFile, SpecContent
-from rebasehelper.plugins.spec_hooks.typo_fix import TypoFix
-from rebasehelper.plugins.spec_hooks.pypi_url_fix import PyPIURLFix
-from rebasehelper.plugins.spec_hooks.escape_macros import EscapeMacros
-from rebasehelper.plugins.spec_hooks.replace_old_version import ReplaceOldVersion
-from rebasehelper.plugins.spec_hooks.paths_to_rpm_macros import PathsToRPMMacros
+from rebasehelper.tests.conftest import SPEC_FILE
 
 
 class TestSpecFile:
     NAME: str = 'test'
     VERSION: str = '1.0.2'
+    SPEC_FILE: str = SPEC_FILE
```

Comment:
This feels really unnecessary, imported `SPEC_FILE` can be used directly, can it not?

Reply:
You are right. Now that I am looking at it, `TestSpecFile.SPEC_FILE` is only used in one test, it will definitely be better to use `SPEC_FILE` directly